ReactWindows/ReactNative.Interfaces/Bridge/IGenericDelegate.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace ReactNative.Bridge
 {
-    interface IGenericDelegate
+    public interface IGenericDelegate
```

Comment:
>IGenericDelegate [](start = 21, length = 16)

This is an implementation detail of our INativeModule base class implementation. We don't need this interface.

Reply:
#fixed